build-system/pr-check/utils.js

```diff
@@ -0,0 +1,174 @@
+
+  const branchPoint = gitMergeBaseMaster();
+  console.log(fileLogPrefix, 'Commit log since branch',
+      colors.cyan(gitBranchName()), 'was forked from',
+      colors.cyan('master'), 'at', colors.cyan(shortSha(branchPoint)) + ':');
+  console.log(gitDiffCommitLog() + '\n');
+}
+
+/**
+ * Starts connection to Sauce Labs after getting account credentials
+ * @param {string} functionName
+ */
+function startSauceConnect(functionName) {
+  process.env['SAUCE_USERNAME'] = 'amphtml';
+  process.env['SAUCE_ACCESS_KEY'] = getStdout('curl --silent ' +
+      'https://amphtml-sauce-token-dealer.appspot.com/getJwtToken').trim();
+  const startScCmd = 'build-system/sauce_connect/start_sauce_connect.sh';
+  const fileLogPrefix = colors.bold(colors.yellow(`${functionName}:`));
+  console.log('\n' + fileLogPrefix,
+      'Starting Sauce Connect Proxy:', colors.cyan(startScCmd));
+  execOrDie(startScCmd);
+}
+
+/**
+ * Stops connection to Sauce Labs
+ * @param {string} functionName
+ */
+function stopSauceConnect(functionName) {
+  const stopScCmd = 'build-system/sauce_connect/stop_sauce_connect.sh';
+  const fileLogPrefix = colors.bold(colors.yellow(`${functionName}:`));
+  console.log('\n' + fileLogPrefix,
+      'Stopping Sauce Connect Proxy:', colors.cyan(stopScCmd));
+  execOrDie(stopScCmd);
+}
+
+/**
+ * Starts a timer to measure the execution time of the given function.
+ * @param {string} functionName
+ * @param {string} fileName
+ * @return {DOMHighResTimeStamp}
+ */
+function startTimer(functionName, fileName) {
+  const startTime = Date.now();
+  const fileLogPrefix = colors.bold(colors.yellow(`${fileName}:`));
+  console.log(
+      '\n' + fileLogPrefix, 'Running', colors.cyan(functionName) + '...');
+  return startTime;
+}
+
+/**
+ * Stops the timer for the given function and prints the execution time.
+ * @param {string} functionName
+ * @param {DOMHighResTimeStamp} startTime
+ * @return {number}
+ */
+function stopTimer(functionName, startTime) {
+  const endTime = Date.now();
+  const executionTime = new Date(endTime - startTime);
+  const mins = executionTime.getMinutes();
+  const secs = executionTime.getSeconds();
+  const fileLogPrefix = colors.bold(colors.yellow(`${functionName}:`));
+  console.log(
+      fileLogPrefix, 'Done running', colors.cyan(functionName),
+      'Total time:', colors.green(mins + 'm ' + secs + 's'));
+}
+
+/**
+ * Executes the provided command and times it. Errors, if any, are printed.
+ * @param {string} cmd
+ * @return {<Object>} Process info.
+ */
+function timedExec(cmd) {
+  const startTime = startTimer(cmd);
+  const p = exec(cmd);
+  stopTimer(cmd, startTime);
+  return p;
+}
+
+/**
+ * Executes the provided command and times it. The program terminates in case of
+ * failure.
+ * @param {string} cmd
+ * @param {string} fileName
+ */
+function timedExecOrDie(cmd, fileName = 'utils.js') {
+  const startTime = startTimer(cmd, fileName);
+  execOrDie(cmd);
+  stopTimer(fileName, startTime);
```

Comment:
This should be `stopTimer(cmd, startTime)`

See https://travis-ci.org/ampproject/amphtml/jobs/493484525#L713-L717